### ISSUE TITLE
FFZ Donor Badge support

### ISSUE THREAD

**Issue by sunkhaskasis** [lay295/TwitchDownloader#103]
Seeing that the new version of Chatterino has support for the FrankerFaceZ badge, I was wondering if you could try adding it to the Twitch Downloader as well?

**Comment by goldbattle** [comment]
I think we will only support twitch badges at this point, might be something to consider in the future.

**Comment by ScrubN** [comment]
I agree we should consider it in the future, so the issue should remain open
